apps/devs-who-run-vue/src/app/core/services/auth.service.ts

```
@@ -0,0 +1,61 @@
+      return null;
+    }
+  }
+
+  static async logout(): Promise<void> {
+    await axios.post(`${this.API_URL}/logout`);
+  }
+
+  static async sendResetEmail(email: string): Promise<void> {
+    await axios.post(`${this.API_URL}/forgot-password`, { email });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)